.gitpod.yml

```diff
@@ -0,0 +1,35 @@
+github:
+  prebuilds:
+    # enable for the default branch (defaults to true)
+    master: true
+    # enable for all branches in this repo (defaults to false)
+    branches: true
+    # enable for pull requests coming from this repo (defaults to true)
+    pullRequests: true
+    # add a "Review in Gitpod" button as a comment to pull requests (defaults to false)
+    addComment: true
+    # configure whether Gitpod registers itself as a status check to pull requests
+    addCheck: false
+
+tasks:
+  - init: |
+      ./scripts/start.sh && ./scripts/stop.sh
+    command: |
+      curl -L --http1.1 https://cnfl.io/ccloud-cli | sudo sh -s -- -b /usr/local/bin
+      echo "🚀 You can now follow steps in https://docs.confluent.io/platform/current/tutorials/cp-demo/docs/overview.html"
+
+vscode:
+  extensions:
+    - ms-azuretools.vscode-docker
+
+ports:
```

Comment:
I have no idea why, but I'm getting notifications for all the ports

Reply:
I do see the same, need to investigate why

Reply:
I can reproduce same issue as reported here https://github.com/gitpod-io/gitpod/issues/1484
I'll go ahead and define ports one by one

Reply:
With the modification, I only see this now:

![CleanShot 2021-10-06 at 12 12 10@2x](https://user-images.githubusercontent.com/4061923/136183885-57b4fcb1-a64c-4ad1-b59b-0b8ee2b8b5a1.jpg)

And same for Kibana